build/common/installer/scripts/fluent-bit-conf-customizer.rb

```diff
@@ -84,6 +84,13 @@ def substituteResourceOptimization(resourceOptimizationEnabled, new_contents)
   return new_contents
 end
 
+def substituteNetworkFlowLogs(networkFlowLogsEnabled, new_contents)
```

Comment:
Since high log scale mode is pre-requisite for Network flow logs. We should set high log scale mode if the Network flow logs enabled. We would need same change for Multi-tenancy logs feature also. Lets discuss about how to make this change

Reply:
There are 2 possible ways 1. if networkFlowLogsEnabled, in docker provider code, we can configure highlogscale mode  or 2.  in AKS RP, if networkFlowLogsEnabled, set the ENABLE_HIGH_LOG_SCALE_MODE=true.                         
can you explore option 1.

Reply:
added in main.sh